Api/Data/FilmInterface.php

```diff
@@ -1,39 +1,21 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Wtc\Film\Api\Data;
 
 interface FilmInterface
 {
-    /**
-     * @return int
-     */
```

Comment:
Unfortunately in an Interface you do need the docblocks:

> For SOAP and REST to work correctly, the following rules must be followed by the service interface's doc block:
>
> All methods exposed by the web API must follow these rules
> 
> All methods on objects expected as parameters or returned must follow these rules
> 
> Parameters must be defined in the doc block as
> 
> @param type $paramName
> Return type must be defined in the doc block as
> 
> @return type
> Valid scalar types include: mixed (or anyType), bool (or boolean), str (or string), integer (or int), float, and double.
> 
> Valid object types include a fully qualified class name or a fully qualified interface name.
> 
> Any parameters or return values of type array can be denoted by following any of the previous types by an empty set of square brackets []
> 

See more: https://developer.adobe.com/commerce/php/development/components/web-api/services/